config/locales/wagon.de.yml

```diff
@@ -1350,6 +1358,9 @@ de:
         mv_yearly_invoice: "MV-Jahresrechnung"
         zusatzsektion_eintrittsrechnung: "Zusatzsektions-Eintrittsrechnung Sektion"
 
+    course_invoices:
+      no_invoice_possible: "WIP Probleme beim Erstellen der Rechung"
```

Comment:
No real error message is specified in #1008. @codez  which error messages are possible for course invoices

Reply:
Habe mal das WIP rausgenommen...